### ISSUE TITLE
Exported contacts from nextcloud creates multiple lines of PRODID in same element, which fails during Import

### ISSUE THREAD

**Issue by devz3r0** [nextcloud/contacts#4158]
### Describe the bug

I am exporting my 78 Contacts from Account A and importing into account B. Most are imported however 2 failed.
During investigation of the logs, the 2 failed with: Validation error in vCard: PRODID MUST NOT appear more than once in a VCARD component

### Steps to reproduce

    1. Go into account A and export the contacts from the nextcloud
    2. go into account B on the same server and import that just exported file
    3. witness the error.

### Expected behavior

all contacts are imported from the just exported file.

### Actual behavior

Validation error in vCard: PRODID MUST NOT appear more than once in a VCARD component

the output in thee file from a working contact entry looks like:

```
BEGIN:VCARD
VERSION:4.0
PRODID:+//IDN bitfire.at//DAVx5/4.3.5.1-gplay ez-vcard/0.12.0
UID:104608ad-dfd8-47f4-a887-c78eebbb0074
FN:<REDACTED>
N:<REDACTED>
TEL;TYPE=cell:+<REDACTED>
REV:20230802T142130Z
END:VCARD
```
one of the 2 errornous entries:
```
BEGIN:VCARD
VERSION:3.0
VERSION:3.0
VERSION:3.0
VERSION:3.0
VERSION:3.0
PRODID:-//ownCloud//NONSGML Contacts 0.5.0.0//EN
PRODID:-//Sabre//Sabre VObject 3.4.7//EN
PRODID:-//ownCloud//NONSGML Contacts 0.5.0.0//EN
N:<REDACTED>
FN:<REDACTED>
TEL;TYPE=VOICE,CELL:+<REDACTED>
REV:2015-12-28T10:51:09+00:00
UID:1030bc3a-e24b-41ee-a2ea-26ed2cf9e32b
END:VCARD
``` 

### Contact version

6.0.0

### Operating system

ubuntu 22.04

### PHP engine version

PHP 8.3

### Web server

Apache (supported)

### Database

MySQL

**Comment by DerDreschner** [comment]
Hello @devz3r0, thanks for taking time to create the issue! I tried to reproduce the issue, but I'm unable to do so. Can you please tell me if the bug is still there on your instance? Thanks!

**Comment by devz3r0** [comment]
Hi, 

Yes I still have multiple lines of PRODID inside one contact. However it is still only at two entries.

**Comment by DerDreschner** [comment]
Yeah, then these entries are broken... We'll have a look to handle that case in the future.

**Comment by devz3r0** [comment]
The problem is, that I can't say if the phone app broke it and the nextcloud did let it slip, if you sync it via caldav, or it happened during the sync itself.
